app/build.gradle

```diff
@@ -29,6 +31,10 @@ android {
     kotlinOptions {
         jvmTarget = '1.8'
     }
+
+    buildFeatures{
```

Comment:
Submit as viewBinding = true

Reply:
it is not necessary since viewbinding would exceed its access rights

Reply:
It's ok